.gitignore

```diff
@@ -1,5 +1,4 @@
 node_modules/
-dist/
```

Comment:
@mesqueeb Why are you adding the dist folder?

Reply:
I can't remember! 😅